### PR TITLE
Online Version 

### PR DESCRIPTION
**Pull request by cvalencia1991**
# In this milestone, I did the following activities:

 -Create the hosting with Github Pages

-Add the link into the README.md file 

 -Tested linters file to lint my HTML and CSS file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,20 +2,13 @@
...
- -Updated CSS file with responsive design to fit desktop version
-
- -Added extra vector images for the desktop version
-
- -Added linters file to lint my html and css file
  
  ## Live Demo 
-- [live portfolio demo](https://cvalencia1991.github.io/Portafolio/)
+ 
+- [live portfolio demo]( https://cvalencia1991.github.io/Portafolio/)
```
Comment (danmainah) [inline README.md:13]: - [ ] Kindly move your authors' title to be above all the authors currently you have one author above the title.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @cvalencia1991, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 1 above)
Comment (cvalencia1991) [reply README.md:13]: ok I'm going to delete the other authors

**Review by mwanawabangona — APPROVED** [review]
Hi @cvalencia1991 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Deployed successfully !! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**